### ISSUE TITLE
view_channel permission, read_messages permission

### ISSUE THREAD

**Issue by BlckViper** [Pycord-Development/pycord#884]
### Summary

read_messages permission is outdated and included in the view_channel 

### Reproduction Steps

- Read the [Official Discord Docs](https://discord.com/developers/docs/topics/permissions#permissions) and search for the `read_messages` permission -> Doesn't exist anymore, cause `view_channel` permission includes the permission of reading messages.
  
- Goto a discord server
  - Open server settings
    - Open the permissions overview and search the `read_messages` permission -> Doesn't exist
  - Open the permissions overview from a text channel and search the permission `read_messages` -> Doens't exist
___ 

```py
import discord
from discord.ext import commands

TOKEN = 'YOUR_DISCORD_BOT_TOKEN'
PREFIX = '!'
INTENTS = discord.Intents.all()

bot = commands.Bot(command_prefix=PREFIX, intents=INTENTS)

@bot.command()
async def test(ctx):
    member: discord.Member = ctx.author
    [print(a) for a in member.guild_permissions]

bot.run(TOKEN)
```

```py
import discord
print([a for a in discord.Permissions.all()])
```

### Minimal Reproducible Code

```python
print([a for a in member.guild_permissions])

print([a for a in discord.Permissions.all()])
```

### Expected Results

```py
>>> (view_channel, True)
```
or
```py
>>> (view_channel, False)
```
___

The permission `read_messages` is outdated, so `read_message` is not allowed to be shown there.
___ 
The permission `view_channel` has to be show there, cause of this is the updated permission. -> [Official Discord Docs](https://discord.com/developers/docs/topics/permissions#permissions)

### Actual Results

```py
>>> (read_messages, True)
```
or 
```py
>>> (read_messages, False)
```
___
- The `view_channel` permission is not listed in:
  - `discord.Permissions.all()`
  - `member.guild_permissions`
- The `read_messages` permission is listed in:
  - `discord.Permissions.all()`
  - `discord.Permissions`
  - `member.guild_permissions`

### Intents

discord.Intents.all()

### System Information

- Python v3.10.0-final
- py-cord v2.0.0-beta
    - py-cord pkg_resources: v2.0.0b4877+g98988f49
- aiohttp v3.7.4.post0
- system info: Windows 10 10.0.22000

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

![grafik](https://user-images.githubusercontent.com/57014404/151660714-e07d8f46-420c-43a3-8c09-4ed53952771b.png)

- The permission `read_messages` is removed from the [Official Discord Docs](https://discord.com/developers/docs/topics/permissions#permissions)
- The permission to read messages is now included into the `view_channel` permission -> [Official Discord Docs](https://discord.com/developers/docs/topics/permissions#permissions)

- Py-Cord API
  - `read_messages` permission is outdated, and not allowed to be shown in the 
    - `discord.Permissions`
    - `discord.Permissions.all()`
    - `member.guild_permissions`

  - `view_channel` permission is updated in the [Official Discord Docs](https://discord.com/developers/docs/topics/permissions#permissions) and therefore the pycord api needs to update:

    - `view_channel` permission needs to be added to:
      - `discord.Permissions.all()`
      - `member.guild_permissions`
    - `read_messages` permission needs to be removed from: 
      - `discord.Permissions.all()`
      - `discord.Permissions`
      - `member.guild_permissions`

- PyCord-Docs
  - `read_messages` permission is outdated, and no longer allowed to be shown in the [Docs for PyCord](https://docs.pycord.dev/en/master/api.html?highlight=read_messages#discord.Permissions.read_messages) -> Add the removal to the PyCord [Changelog](https://docs.pycord.dev/en/master/whats_new.html)
  - Update the description of the `view_channel` permission

**Comment by plun1331** [comment]
This got fixed in #881, and is pretty much a duplicate of #878

**Comment by Lulalaby** [comment]
Read messages is an alias in pycord.

Do not think about read message history.

Look into the view channel permission table on the docs and you see the note that that gives the capability to read messages.

Closing as invalid and duplicate
